op.hcl

```diff
@@ -0,0 +1,32 @@
+// https://developer.1password.com/docs/cli/
+// https://app-updates.agilebits.com/product_history/CLI2
+description = "1Password CLI allows you to automate administrative tasks, securely provision secrets across development environments, and use biometrics to authenticate in the terminal."
+binaries = ["op"]
+test = "op --version"
+
+darwin {
+    source = "https://cache.agilebits.com/dist/1P/op2/pkg/v${version}/op_apple_universal_v${version}.pkg"
+
+    on "unpack" {
+        // How "${HOME}/usr/local/bin/op" was determined:
+        // 1. curl https://cache.agilebits.com/dist/1P/op2/pkg/v2.18.0/op_apple_universal_v2.18.0.pkg --output op_v2.18.0.pkg
+        // 2. pkgutil --expand op_v2.18.0.pkg op-v2.18.0/
+        // 3. grep install-location op-v2.18.0/op.pkg/PackageInfo
+        rename {
+            from = "${HOME}/usr/local/bin/op"
+            to = "${root}/op"
+        }
+
+        // clean up directories created by pkg install-location
+        delete {
```

Comment:
local testing shows the correct expected behavior:

```sh
debug: installer: The upgrade was successful.
debug:op-2.19.0: mv '/Users/mbeatty/usr/local/bin/op' '/Users/mbeatty/Library/Caches/hermit/pkg/op-2.19.0/op'
debug:op-2.19.0: rm -r /Users/mbeatty/usr/local/bin/
debug:op-2.19.0:exec: /Users/mbeatty/Library/Caches/hermit/pkg/op-2.19.0/op --version
debug: 2.19.0
```

Reply:
on second thought, this could be unintentionally destructive so just going to `message`